Tests/IntegrationTests/IntegrationTests.swift

```diff
@@ -2,32 +2,32 @@ import Foundation
 import XCTest
 import MySQL
 
-  var connectionString: MySQLConnectionString?
+var connectionString: MySQLConnectionString?
 
 public class IntegrationTests: XCTestCase {
   public override func setUp() {
-    let mySQLServer = ProcessInfo.processInfo.environment["MYSQL_SERVER"]
-    connectionString = MySQLConnectionString(host: mySQLServer!)
+    let mySQLServer = "0.0.0.0"
```

Comment:
This line won't work, it assumes that the server is running on the same host as the test which it may not be